1-Draft/RFCNNN-WindowsPS-Compatibility-Utilities.md

```diff
@@ -0,0 +1,106 @@
+---
+RFC:
+Author: Andrew Menagarishvili
+Status: Draft
+SupercededBy:
+Version: 0.1
+Area: Language
+Comments Due: 10/10/2019
+Plan to implement: Yes
+---
+
+# Windows PowerShell Compatibility Utilities
+
+One of the big factors preventing existing Windows PowerShell users from moving to PowerShell Core has been cmdlet coverage. Specifically being able to use existing Windows PowerShell modules in PowerShell Core. Ease of UX is important here - working with 'WindowsPS-only' modules from PS Core should not be any different than working with 'PS Core - compartible' modules. Also this should not require any setup from the user.

```

Comment:
It might be good to mention that the non-compatibility is due to the different dotNet runtimes used between Windows PowerShell and PowerShell Core.

Reply:
Added.